assets/css/desktop.css

```diff
@@ -0,0 +1,464 @@
+html {
+  background-color: #fff;
+  font-family: 'Poopins', sans-serif;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0 !important;
+  padding: 0 !important;
+  background-color: #f7f7f9;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+/** Nav section start */
+
+.menu {
+  display: none;
+}
+
+.bar-section {
+  width: 100%;
+  background-color: white;
+  position: fixed;
+}
+
+.wrapper {
+  height: 50px;
+  margin-left: 155px;
+  margin-right: 155px;
+}
+
+.nav-bar {
+  padding-top: 5px;
+}
+
+.nav-bar ul {
+  display: flex;
+  align-self: center;
+  list-style: none;
+  justify-content: space-between;
+}
+
+.nav-bar a {
+  text-decoration: none;
+}
+
+.logo {
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 20px;
+  flex: 10;
+}
+
+.logo a {
+  color: #6070ff;
+}
+
+.menu-desktop a {
+  color: #344563;
+}
+
+.menu-desktop {
+  flex: 1;
+  font-size: 15px;
+  font-weight: 500;
+  align-content: center;
+  padding-right: 12px;
+}
+
+/*** Nav section end **/
+
+/** Headline section start **/
+
+.headline-wrapper {
+  background-image: url("../images/header_section_background_desktop.png");
+  background-size: contain;
+  background-position: center;
+  background-repeat: no-repeat;
+  border-bottom-left-radius: 110px;
+  background-color: #fff;
+  height: 820px;
+  display: flex;
+  align-items: center;
+}
+
+.headline-container {
+  display: flex;
+  flex-direction: column;
+  align-self: center;
+  justify-items: center;
+  margin-left: 260px;
+  margin-right: 260px;
+}
+
+.intro-text {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 60px;
+  font-family: 'Poppins', sans-serif;
+  color: #091e42;
+  margin: 12px 0;
+}
+
+.intro-message {
+  font-family: 'Poppins', sans-serif;
+  font-size: 18px;
+  color: #344563;
+}
+
+.connect h3 {
+  color: #6070ff;
+}
+
+.social {
+  display: flex;
+  padding-left: 0;
+  list-style: none;
+  width: 200px;
+  margin: 0 25px 0 0;
+}
+
+.social-media {
+  font-size: 20px;
+  color: #505f79;
+  flex: 1;
+}
+
+.social-media a {
+  color: #505f79;
+}
+
+/*** Headline section End ***/
+
+/*** Works section **/
+
+.works-flex {
+  background: white;
+  margin: 142px;
+  border-radius: 8px;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+}
+
+.start {
+  order: 1;
+}
+
+.end {
+  order: 2;
+}
+
+.img-style {
+  display: none;
+}
+
+.img-style-desktop {
+  display: block;
+}
+
+.work-image {
+  align-self: center;
+  border-radius: 10px;
+  margin: 24px;
+}
+
+/* Text */
+
+.intro-text-works {
+  font-weight: 650;
+  padding-top: 10px;
+  font-size: 18px;
+  font-family: 'Poppins', sans-serif;
+  color: #344563;
+  margin: 0 0 0 0;
+}
+
+.titles-container {
+  display: flex;
+  flex-direction: row;
+  padding-left: 0;
+  justify-self: flex-start;
+  padding-top: 10px;
+  width: 100%;
+}
+
+.titles-item {
+  font-family: 'Poppins', sans-serif;
+  font-size: 16px;
+  color: lightslategray;
+  font-weight: 500;
+  list-style-image: url("../images/Counter.png");
+  flex: 2;
```

Comment:
- [ ] I believe that this is causing your items to align weirdly. You should consider other alternatives, in order to match the original design better, which has all elements aligned to the left.

Figma design:
![image](https://user-images.githubusercontent.com/66967274/144153908-449836ff-0656-4ff9-b75c-a85c05afdc90.png)

Your design:
![image](https://user-images.githubusercontent.com/66967274/144153975-99524939-3028-43f4-ba88-c5810dea8804.png)